tests/test_sequence.py

```diff
@@ -220,7 +226,7 @@ def test_reasonable_predictions(self):
         self.assertTrue(any(pred["text"].strip() == "dog" for pred in predictions[0]))
 
     def test_chunk_long_sequences(self):
-        test_sequence = ["I am a dog. A dog that's incredibly bright. I can talk, read, and write!" * 10]
+        test_sequence = ["I am a dog. A dog that's incredibly bright. I can talk, read, and write! " * 10]
```

Comment:
Im intrigued whether this space is required and if so why?

Reply:
I don't think it's required I think I just noticed that it was going to smush words together.  Let me confirm test still passes w/o space.

Reply:
Oh totally, I missed the position of the * 10 there.